labels.py

```diff
@@ -0,0 +1,26 @@
+from cProfile import label
+from lib2to3.pytree import LeafPattern
+from Labelling.config.constants import TEST_IMG_DIR_PATH
```

Comment:
I deprecated the use of TEST_IMG_DIR_PATH. You now select the image folder with the GUI

This would be a bigger change (and I could help you with this), but we could actually add this to the GUI (e.g. we could make it so that in the top menu, you click an option called "Export Stuff", and then you select the directories... this is something we can do later and I'll let you file the issue if you'd like 😄)!